AutomatedTesting/Gem/PythonTests/Atom/TestSuite_Main_GPU.py

```diff
@@ -226,9 +226,9 @@ def test_MaterialEditorLaunch_AllRHIOptionsSucceed(
             self, request, workspace, project, launcher_platform, generic_launcher, exe_file_name, cfg_args):
         """
         Tests each valid RHI option (Null RHI excluded) can be launched with the MaterialEditor.
-        Checks for the "Finished loading viewport configurtions." success message post lounch.
+        Checks for the "Finished loading viewport configurations." success message post launch.
         """
-        expected_lines = ["Finished loading viewport configurtions."]
+        expected_lines = ["Finished loading viewport configurations."]
         unexpected_lines = [
             # "Trace::Assert",
```

Comment:
Nitpick: I'd either remove these lines, or add a comment explaining why they're temporarily disabled.

Reply:
There is a ticket for it, I will link you